drivers/gles2/rasterizer_storage_gles2.cpp

```diff
@@ -6133,7 +6140,9 @@ void RasterizerStorageGLES2::initialize() {
 			glBindTexture(GL_TEXTURE_2D, 0);
 			glDeleteTextures(1, &depth);
 		}
+#ifndef JAVASCRIPT_ENABLED
```

Comment:
That's not the closing bracket corresponding to `if (status != GL_FRAMEBUFFER_COMPLETE) {`.

Reply:
Whoops. That's embarrassing. Fixed now.

Reply:
Discovered another problem with this PR but think I fixed it (glTexImage2D doesn't accept GL_DEPTH_COMPONENT(X) enums on webgl 1.0). Don't want to waste anyone's time while I verify though. Should I convert to draft while I test?

Reply:
Amended PR with what I think are final fixes. Tested on

-Windows Desktop
-Chrome Windows
-Chrome Android
-Safari IOS

No errors in dev console on Chrome Windows.